### ISSUE TITLE
Will you add support for SCM?

### ISSUE THREAD

**Issue by jambonmcyeah** [itsFrank/MinecraftHDL#4]
https://minecraft.curseforge.com/projects/super-circuit-maker
Please add support for this mod because it will make designing circuits much easier

**Comment by itsfrank** [comment]
Yes this was proposed on reddit and its really good step forward in allowing for much larger circuits, I will have to study the SCM api when I have the time. I'm currently doing a Master's so time is not something I have much of unfortunately
